### PR TITLE
Feature/speaker level

### PR DESCRIPTION
**Pull request by JoeJasinski**
This is the PR to complete issue https://github.com/chicagopython/chipy.org/issues/118

This adds a dropdown on the talk proposal page to indicate talk level (novice, intermedia, and advanced). 

Deployment instructions.  A migration needs to be run to deploy this.

```
heroku run --app chipy python ./manage.py migrate meetings
```

### CODE REVIEW

**Comment by emperorcezar** [comment]
Would we want to display this on the meeting list?

**Comment by emperorcezar** [comment]
Also, maybe instead of `python_level` it could be `experience_level`. I can see a data science talk that has simple python, but difficult math.

**Comment by JoeJasinski** [comment]
I pulled the python_level name from the pycon website, but I like the experience_level wording a better.  Thanks for the feedback; I'll make that change.

**Comment by JoeJasinski** [comment]
@emperorcezar I made the change to the field name.

File: chipy_org/apps/meetings/migrations/0003_topic_python_level.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('meetings', '0002_rsvp_guests'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='topic',
+            name='python_level',
```
Comment (emperorcezar) [inline chipy_org/apps/meetings/migrations/0003_topic_python_level.py]: This is still `python_level`

**Comment by emperorcezar** [comment]
You need to squash the migrations.

**Comment by emperorcezar** [comment]
Also. We need to display this on the front end somewhere.

**Comment by JoeJasinski** [comment]
I can squash

**Comment by JoeJasinski** [comment]
This was mainly to be used as an aid to help us choose a good mix of talks and is visible through the admin.  However, it wouldn't hurt to make this end user facing as well.  Where do you suggest a good place to include it is?

**Comment by emperorcezar** [comment]
Ah.
I'd say right after the author line.

**Comment by JoeJasinski** [comment]
I've added the experience level to the front end and I've reset the migrations.  (Note, if you have a local copy and have migrated past migration "meetings" "0002", you need to migrate back to 0002, checkout the code, checkout the code changes, and migrate again)

**Comment by emperorcezar** [comment]
👍

**Comment by emperorcezar** [comment]
@JoeJasinski If you fix the conflicts, I approved this, so you can merge it.

**Comment by JoeJasinski** [comment]
Thanks, I'll take a look at that.

**Comment by JoeJasinski** [comment]
@emperorcezar, I've resolved the merge conflict.  Let me know if it looks good.

**Pull request merged**